package.json

```diff
@@ -24,9 +24,11 @@
     }
   },
   "dependencies": {
-    "@openpolitica/matomo-next": "1.2.1",
+    "@datapunt/matomo-tracker-react": "0.4.0",
     "chrome-aws-lambda": "10.0.0",
     "classnames": "2.3.1",
+    "install": "0.13.0",
+    "npm": "7.19.0",
```

Comment:
Did `npm` and `install` end up here by accident?

Reply:
They did! Good catch! I will remove them!